### PR TITLE
Fixes 14867: advise to remove ovirt_provision_plugin

### PR DESCRIPTION
**Pull request by helge000**
I put it right in upgrading...

### CODE REVIEW

File: _includes/manuals/1.11/3.6_upgrade.md [hunk 1]
```diff
@@ -7,6 +7,19 @@ When upgrading across more than one version it is recommended to upgrade to each
 intermediate version and follow all upgrade instructions for the previous versions
 before updating to {{page.version}}.
 
+#### Known issues
+Currently, [tfm-rubygem-ovirt_provision_plugin prevents](https://github.com/theforeman/ovirt_provision_plugin/issues/5) 
+the upgrade process to finish correctly and needs to be removed if it was installed 
+previously in Foreman versions <= 1.10.X:
```
Comment (domcleal) [inline _includes/manuals/1.11/3.6_upgrade.md:13]: add "on EL6 or EL7.", as the instructions here are specifically for it and no package is provided for Debian/Ubuntu

File: _includes/manuals/1.11/3.6_upgrade.md [hunk 2]
```diff
@@ -7,6 +7,19 @@ When upgrading across more than one version it is recommended to upgrade to each
...
+
+    foreman-installer --no-enable-foreman-plugin-ovirt-provision
+    yum remove tfm-rubygem-ovirt_provision_plugin
+
+Continue with the steps below.
+
+*Note*: ovirt_provision_plugin does not handle oVirt compute resources but deployment of oVirt
+notes from within oVirt.
```
Comment (domcleal) [inline _includes/manuals/1.11/3.6_upgrade.md]: "nodes"

**Comment by helge000** [comment]
Added your remarks, @domcleal

**Comment by domcleal** [comment]
Merged, thanks @helge000.

**Pull request merged**